openlibrary/templates/books/edit/edition.html

```diff
@@ -91,7 +97,11 @@
     \$("#works").setup_multi_input_autocomplete(
         "input.work-autocomplete",
         render_work_field,
-        { endpoint: "/works/_autocomplete" },
+        {
+            endpoint: "/works/_autocomplete",
+            addnew: $cond(is_privileged_user, 'true', 'false'),
+            new_name: '$_('__new__')',
```

Comment:
@cdrini just noticed this, why is this wrapped in `$_()` it's only going to try to localise this system string? I don't think that's a good idea... is this doing anything useful?

Reply:
It's just dummy text; it doesn't actually affect anything (it's just what's displayed when you select "new"). So I i18n-ed mostly cause why not?

Reply:
Marking as won't fix.

Reply:
So this doesn't need to match the literal `__new__` instances in the code? In that case, I'd recommend making it something less cryptic like "Create new work" or something else that makes sense to the user in context.

Reply:
fixed